common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -328,7 +330,8 @@ class @Problem
   reset_internal: =>
     Logger.log 'problem_reset', @answers
     $.postWithPrefix "#{@url}/problem_reset", id: @id, (response) =>
-        @render(response.html)
+        reset_error = response['error']
```

Comment:
@mehrozezahid better way to access an object's attribute is like `response.error`